node_modules/oae-zencoder/config/config.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2016 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var Fields = require('oae-config/lib/fields');
+
+module.exports = {
+    'title': 'OAE Zencoder Module',
+    'zencoder': {
+        'name': 'Zencoder Configuration',
+        'description': 'Configuration for Zencoder video transcoding',
+        'elements': {
+            'apiKey': new Fields.Text('API Key', 'The Zencoder API key for your account', '', {'suppress': true, 'globalAdminOnly': true}),
+            'amazons3-bucket': new Fields.Text('Amazon S3 Bucket', 'The Amazon S3 bucket for output videos', '', {'suppress': true, 'globalAdminOnly': true}),
+            'amazons3-access-key': new Fields.Text('Amazon Access Key', 'Your Amazon Access key', '<access-key>', {'suppress': true, 'globalAdminOnly': true}),
+            'amazons3-secret-key': new Fields.Text('Amazon Secret Key', 'Your Amazon Secret key', '<secret-key>', {'suppress': true, 'globalAdminOnly': true}),
+            'amazons3-region': new Fields.Text('Amazon S3 Region', 'The region for your S3 bucket', 'eu-west-1', {'suppress': true, 'globalAdminOnly': true})
+        }
+    }
+};
```

Comment:
When we do make it viewable by a tenant, should we keep the ability to make it explicitly "enabled" / "disabled" in configuration? I think that's missing from what we had in the mediacore config.